### PR TITLE
add: note on how to use hook post data + complete event list

### PR DESCRIPTION
**Pull request by serjoscha87**

### CODE REVIEW

File: api/webhooks/index.md [hunk 1]
```diff
@@ -23,10 +23,63 @@ Webhooks allow you to specify a URL to which you would like Cockpit to post data
 
 All data will be send as _application/json_.
 
+### Using POST-data in the webhook destination script
 
-### Useful events
+Some hooks send data along with the hook (like some of the collection event hooks which send the fresh new collection's name and confifuration).
```
Comment (serjoscha87) [inline api/webhooks/index.md]: typo 🤦‍♂️ "confifuration"

File: api/webhooks/index.md [hunk 2]
```diff
@@ -23,10 +23,63 @@ Webhooks allow you to specify a URL to which you would like Cockpit to post data
...
 
+The data is posted to the webhook destinatin script in RAW-Post format. So the global var _**$\_POST**_ won't contain any data.
```
Comment (serjoscha87) [inline api/webhooks/index.md]: another typo "destinatin"

what the heck did I type here?! Should I cancel the PR in order to correct typos or would you instead correct them?

Comment (aheinze) [reply api/webhooks/index.md]: Just do a new commit in you forked branch and it will be added to the PR.

**Comment by serjoscha87** [comment]
okay; I removed the typos and changed the post text a bit

**Comment by aheinze** [comment]
Thanks!

**Pull request merged**